README.md

```diff
@@ -1,2 +1,5 @@
 # HTA-hub
 Website for a repository/inventory of health economics R packages 
+
+# Quarto render command
+quarto render
```

Comment:
According to [https://github.com/quarto-dev/quarto-actions/blob/main/examples/example-01-basics.md](quarto-actions), this command should be run at least once locally (which I have done): `quarto publish gh-pages`

Reply:
[Difference between quarto render and publish](https://github.com/quarto-dev/quarto-cli/discussions/5085)